modyn/selector/internal/selector_strategies/downsampling_strategies/rho_loss_downsampling_strategy.py

```diff
@@ -28,6 +27,11 @@ def __init__(
         maximum_keys_in_memory: int,
     ):
         super().__init__(downsampling_config, modyn_config, pipeline_id, maximum_keys_in_memory)
+        # The choice of the selection strategy does not matter as long as it does not downsample data
+        # at the trainer server. (We don't want to further select on holdout set)
```

Comment:
It does not matter? I think if you e.g. went for random 50% presampling it would matter, wouldn't it? I agree this is the correct strategy to use here but I don't understand why it does not matter

Reply:
The presampling does not matter. Even if we have a random 50% presampling we never put it into use. 

Recall normally how presampling is triggered is by calling `inform_data_and_trigger`, which internally computes and stores the TSS using a SQL query produced by presampler.

In the RHO-LOSS case, the rho pipeline's training data (i.e. holdout set) is hard-code computed by sampling a ratio of trigger data from the main pipeline. The presampling config of the rho pipeline never plays a role.

The downsampling config of the rho pipeline can play a role however (if we set a downsampler), as the training process on the trainer server side for a rho pipeline is identical to a normal pipeline.

Reply:
Ah, that makes sense. I somehow thought you'd inform the selector about the samples you want to use for IL model training again.